### DISCUSSION TITLE
Favicon Not Accessible

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1548] [Q&A] [answered]
I've configured favicon in Head in `.vitepress/config.ts` and pushed all the favicon package in `docs/favicon` directory.

But, it's not getting the work done.

[Here's](https://realfavicongenerator.net/favicon_checker?protocol=https&site=reesors.vercel.app) the analysis link to the Real Favicon Generator

Site Source Code: [GitHub](https://github.com/VedxP/reesors)

**Comment by brc-dd** [comment, marked as answer]
Move contents of `/docs/favicon/` to `/docs/public/`. And instead of `/docs/favicon/...`, write `/...` in the config.

**Reply by vedxp** [reply]
> Move contents of `/docs/favicon/` to `/docs/public/`. And instead of `/docs/favicon/...`, write `/...` in the config.

Thank you so much for helping, sir :)
